drop-in/src/main/res/template/values/strings.xml.tt

```diff
@@ -30,6 +30,9 @@
     <string name="checkout_giftcard_remove_gift_cards_positive_button">%%storedPaymentMethod.disable.confirmButton%%</string>
     <string name="checkout_giftcard_remove_gift_cards_negative_button">%%storedPaymentMethod.disable.cancelButton%%</string>
     <string name="checkout_remove_stored_payment_method_body">%%storedPaymentMethod.disable.confirmation%%</string>
+    <string name="checkout_qr_code_permission_not_granted">%%runtimePermission.permissionDeniedMessage%%</string>
```

Comment:
These 3 strings are generic and can be used later for other components that need storage permission. I think we should remove the `qr_code` part from the string names